lib/react_client.dart

```diff
@@ -429,7 +429,7 @@ _convertEventHandlers(Map args) {
   args.forEach((propKey, value) {
     var eventFactory = _eventPropKeyToEventFactory[propKey];
     if (eventFactory != null && value != null) {
-      args[propKey] = (events.SyntheticEvent e, [String domId, Event event]) => zone.run(() {
+      args[propKey] = (events.SyntheticEvent e, [_, __]) => zone.run(() {
```

Comment:
For my own knowledge @jacehensley-wf - what is the reason for relaxing the type / obfuscating the arguments here?